### PR TITLE
#5179 and TODO about avoid forks

### PR DESCRIPTION
**Pull request by dariosharp**
My solution about issue #5179 and I implemented part of TODO about avoid forks in 'izz*' / 'izzq' / 'izzj'.

### CODE REVIEW

File: libr/core/cmd_info.c [hunk 1]
```diff
@@ -338,28 +343,44 @@ static int cmd_info(void *data, const char *input) {
 					return 0;
 				}
 				biname = r_str_escape (core->file->desc->name);
+				if(!tmpcore){
```
Comment (radare) [inline libr/core/cmd_info.c]: Missing spacs around ()

File: libr/core/cmd_info.c [hunk 2]
```diff
@@ -338,28 +343,44 @@ static int cmd_info(void *data, const char *input) {
...
+	                                eprintf ("Cannot create core\n");
+				        return 0;
+	                        }
+				tmpcore->bin->minstrlen = min;
+				tmpcore->bin->maxstrlen = max;
+				if(!r_bin_load (tmpcore->bin, biname, UT64_MAX, UT64_MAX, xtr_idx, fd, rawstr)){
```
Comment (radare) [inline libr/core/cmd_info.c]: Missing space befoee (

**Comment by Maijin** [comment]
@dariosharp See this https://github.com/radare/radare2/issues?utf8=✓&q=is%3Aissue%20is%3Aopen%20label%3Ajson%20

**Comment by Maijin** [comment]
and https://github.com/radare/radare2/issues?utf8=✓&q=is%3Aissue%20is%3Aopen%20label%3Aeasy%20

**Comment by dariosharp** [comment]
ok thx, I'm going to try to fix one of those

File: libr/core/cmd_info.c [hunk 3]
```diff
@@ -338,28 +343,44 @@ static int cmd_info(void *data, const char *input) {
 					return 0;
 				}
 				biname = r_str_escape (core->file->desc->name);
+				if (!tmpcore) {
+	                                eprintf ("Cannot create core\n");
+				        return 0;
+	                        }
+				tmpcore->bin->minstrlen = min;
+				tmpcore->bin->maxstrlen = max;
+				if (!r_bin_load (tmpcore->bin, biname, UT64_MAX, UT64_MAX, xtr_idx, fd, rawstr)){
```
Comment (radare) [inline libr/core/cmd_info.c]: fd is always -1 here

**Comment by radare** [comment]
There's no need to instantiate RCore to grab info from RBin, it is better than calling rabin2 at the end, but it opens the door to more leaked memory, but thats a problem that we may face soon or late :P and its unrelated to this change.

Also, i dont think we have tests for this.

**Comment by dariosharp** [comment]
Do you think it's may enough use RBin instead of RCore or is the appropriate way completely different?

**Comment by radare** [comment]
I guess that it will enough with rbin. But i can merge this PR for now and we can experiment with only rbin later

**Pull request merged**